build_exercise_proj/tools_lib_proj/CMakeLists.txt

```
@@ -1,4 +1,5 @@
-cmake_minimum_required(VERSION 3.1)
-project(tools_lib_project)
+cmake_minimum_required(VERSION 3.1) 
+project(tools_lib_project) 
 
-add_library(tools tools.cpp)
\ No newline at end of file
+add_library(tools tools.cpp) 
```

add_library creates either a STATIC or SHARED library.
Ommiting the parameter will default to STATIC (whould you are correctly using).
However, it is good practice to explicitly state it.

```suggestion
add_library(tools STATIC tools.cpp) 
```

---

I've missed something here.
It is highly recommend to name your library after the project name.
Thus "tools" should be substituted with ${PROJECT_NAME}